### ISSUE TITLE
Problems installing the package

### ISSUE THREAD

**Issue by pachard** [asheshrambachan/HonestDiD#1]
Hello, 
I tried to download the package following the steps you highlighted but get the following error message 

installation of package ‘.../Local/Temp/RtmpOeI1SA/file418039153388/HonestDiD_0.1.0.tar.gz’ had non-zero exit status

I tried to google the message and find a solution but did not find one. Would you have tips? 
Thanks in advance,

Pascal

**Comment by asheshrambachan** [comment]
Could you give some more information about the error? E.g. what is your OS + R version? Could you attach a screenshot of the error as well?

**Comment by pachard** [comment]
Hi, I am using Windows 10. I updated R before installing the package. When I installed devtools, I was asked if I wanted to update specific packages and had issues with that (in particular the rlang package), so I re-ran the command without updating them and this is when I had the message that I showed you. 
When I googled the message, nothing really came out. I tried the next day, same problem and then I opened the issue. For some reason I tried it again (exactly the same few lines) after you wrote back and everything worked.  
Sorry for the diturbance
